### PR TITLE
Avatar group to allow square overflow

### PR DESCRIPTION
**Pull request by RiteshMakan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/36691

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Produce a prop for the Avatar.Group to accept a shape prop of `circle` or `square`      |
| 🇨🇳 Chinese |     为 Avatar.Group 生成一个 prop 以接受 `circle` 或 `square` 的 shape prop      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
Why change so many files?

图片

**Comment by RiteshMakan** [comment]
> Why change so many files?
> 
> 图片

That was from running `npm test -- -u` to update the snapshots

**Comment by afc163** [comment]
图片

That make ci failed.

**Comment by RiteshMakan** [comment]
@afc163  What can I do to fix it? 
1. I am not sure why when I run `npm test -- -u` to update the snapshots it makes changes to so many files.
2. How do I fix so that ci make passes.

image

Could it be related to my node version: `Node.js v18.2.0`

**Comment by afc163** [comment]
Remove yarn.lock and pacakge-lock.json and reinstall node_modules.

**Comment by RiteshMakan** [comment]
@afc163 I removed the `node-modules` and reinstalled this seems to have worked. Is all ok now?

**Comment by RiteshMakan** [comment]
Is there anything else I need to do here?

**Review by MadCcc — APPROVED** [review]

File: components/avatar/group.tsx [hunk 1]
```diff
@@ -26,7 +27,14 @@ export interface GroupProps {
 
 const Group: React.FC<GroupProps> = props => {
   const { getPrefixCls, direction } = React.useContext(ConfigContext);
-  const { prefixCls: customizePrefixCls, className = '', maxCount, maxStyle, size } = props;
+  const {
+    prefixCls: customizePrefixCls,
+    className = '',
+    maxCount,
+    maxStyle,
+    shape = 'circle',
```
Comment (MadCcc) [inline components/avatar/group.tsx:35]:
```suggestion
   maxShape = 'circle',
```

My bad... `maxShape` is better here, I misunderstood the meaning.

Comment (RiteshMakan) [reply components/avatar/group.tsx:35]: @MadCcc What should I do? Do you want me to push a commit to rename `shape` back to `maxShape`?

Comment (RiteshMakan) [reply components/avatar/group.tsx:35]: Just a follow up @MadCcc. Do you want me to push a commit to rename `shape` back to `maxShape`?

**Comment by afc163** [comment]
Conflicting

**Comment by afc163** [comment]
ping

**Comment by afc163** [comment]
ping

**Comment by afc163** [comment]
ping @RiteshMakan

**Comment by afc163** [comment]
Closing for no response to resolve conflicts, feel free to create a new PR any time.

**Pull request closed without merging**